README.md

```diff
@@ -249,6 +249,7 @@ are designated with a ✅, and hosted projects with a 💙.
 - [Headless Node Renderer](https://github.com/ConservationMetrics/mapgl-tile-renderer) - Headless Node.js MapGL renderer for generating MBTiles with styled raster tiles.
 - [chiitiler](https://github.com/Kanahiro/chiitiler) - chiitiler - "Tiny MapLibre Server" is alternative to Tileserver GL, designed to runs on serverless infrastructures. [demo](https://spatialty-io.github.io/chiitiler-demo/)
 - [TileServer GL](https://github.com/maptiler/tileserver-gl) - Vector tile server from MBTiles archives + server-side rasterizing with MapLibre GL native.
+- [Versatiles](https://versatiles.org/) - A whole toolchain for creating and displaying vecotor maps using the [https://shortbread-tiles.org/schema/](shortbread specification).
```

Comment:
Instead of a single link to versatiles, I would recommend splitting this into different tools for different sections, if those are available as stated.